src/styles/_bootstrap_variables_mapping.scss

```diff
@@ -29,17 +29,17 @@
   --bs-teal: #{$teal};
   --bs-cyan: #{$cyan};
   --bs-primary: #{$primary};
-  --bs-primary-bg: #{$primary-bg};
+  --bs-primary-bg: #{$ds-primary-bg};
   --bs-secondary: #{$secondary};
-  --bs-secondary-bg: #{$secondary-bg};
+  --bs-secondary-bg: #{$ds-secondary-bg};
   --bs-success: #{$success};
-  --bs-success-bg: #{$success-bg};
+  --bs-success-bg: #{$ds-success-bg};
   --bs-info: #{$info};
-  --bs-info-bg: #{$info-bg};
+  --bs-info-bg: #{$ds-info-bg};
   --bs-warning: #{$warning};
-  --bs-warning-bg: #{$warning-bg};
+  --bs-warning-bg: #{$ds-warning-bg};
   --bs-danger: #{$danger};
-  --bs-danger-bg: #{$danger-bg};
+  --bs-danger-bg: #{$ds-danger-bg};
```

Comment:
If we _do_ end up needing dedicated ds versions of these bootstrap background colors, I still wouldn't override the bootstrap css variables with them. The idea behind this mapping is that it is exactly what's in the sass variables.